scripts/codebuild-artifacts.sh

```diff
@@ -23,19 +23,18 @@ function upload_to_s3 {
 # Lambdas
 ############################################
 
-LAMBDAS=$(ls src/lambdas)
+LAMBDAS=$(ls build)
```

Comment:
To avoid listing files, would it be better to list directories only?
e.g.
`ls -d */ | cut -f1 -d'/'`

Reply:
there are only directories in this folder when it builds